### DISCUSSION TITLE
docs/guide/quickStart

### DISCUSSION THREAD

**Discussion by giscus** [iPanelDev/docs#3] [Giscus]
# docs/guide/quickStart

1. 安装运行环境

http://localhost:3000/docs/guide/quickStart

**Comment by kangimc** [comment]
作者大大，我可以提供Twikoo评论区
有需要吗？

**Reply by Zaitonn** [reply]
谢谢，但是暂时没有这方面的考虑

**Reply by kangimc** [reply]
OK

**Comment by chenchen71956** [comment]
作者大大你好，我在下载相关的依赖在win上运行时，我双击它，弹出一个cmd的弹窗，然后什么都没发生，当我登录0.0.0.0:30000（本机ip）时并没有网页出现，也没有401和404的提示，这代表着它没有正常运行，求解

**Reply by Zaitonn** [reply]
打开同目录的设置文件`settings.json`，修改其中的实例密码`instancePassword`使之不为空即可

这个问题已经被修复了但是还没发布新版本

**Reply by chenchen71956** [reply]
感谢您的回复，我成功登录了控制台
